gossip/gossip.go

```diff
@@ -700,9 +700,6 @@ func (g *Gossip) getNextBootstrapAddress() net.Addr {
 		g.resolverIdx %= len(g.resolvers)
 		g.resolversTried[g.resolverIdx] = struct{}{}
 		resolver := g.resolvers[g.resolverIdx]
-		if resolver.IsExhausted() {
```

Comment:
Mind providing a history level on what `IsExhausted` initially accomplished and what value it was providing. I couldn't deduce any from the current code, though I didn't go back to see what the code originally looked like.